.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,29 @@
+---
+name: Bug Report
+about: Report bugs in Azure Go SDK packages.
+---
+
+### Bug Report
+
+<!--
+Thank you for opening a bug report. For faster processing, please include:
+-->
+
+* import path of package in question, e.g. `.../services/compute/mgmt/2018-06-01/mgmt`
+* SDK version, e.g. `master`, `latest`, `18.1.0`
```

Comment:
The branch names might be too vague, we should probably ask for commit hashes if they aren't pinned to a semver.

Reply:
Makes sense, but I'd like to provide succinct instructions on how to find the revision. I added a hint about using `dep status`, is there something else we'd recommend, like running a git command in the local source dir?